### ISSUE TITLE
Allow delegatebw/undelegatebw with no-native EOS blockchain

### ISSUE THREAD

**Issue by DenisCarriere** [eoscanada/eosc#114]
```
$ eosc -u http://<API> system delegatebw <ACCOUNT> <ACCOUNT> "10.0000 SYS" "10.0000 SYS"
ERROR: "network bw stake qty" invalid: EOS has only 4 decimals
```

**Comment by abourget** [comment]
I think the best solution to this would be to default all `asset` fields to `EOS` with precision=4 unless the symbol is specified.

Right now, we have two funcs: `NewEOSAssetFromString` and `NewAsset`. We'd need a third one: `NewEOSDefaultAsset()` or something.

Specifying `2.5` would mean `2.5000 EOS`, whereas specifying `2.5 EOS` would mean specifying an amount of `25` with a precision of `1` and the symbol name `EOS` (which isn't compatible with the native mainnet token).

Would that work?

**Comment by abourget** [comment]
Fixed in 9821c71e36a0cff82821a9842459373ba6c98c24 .. we'll release some time soon

**Comment by DenisCarriere** [comment]
I believe so, as long as "EOS" is not hardcoded and it can be modified somehow by the user settings or auto-detected by `eosc`
